manim/mobject/functions.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         self.function = function
         t_range = np.array([0, 1, 0.01]) if t_range is None else t_range
         if len(t_range) == 2:
-            t_range = [*t_range, 0.01]
+            t_range = np.array([*t_range, 0.01])
```

Comment:
(Non-blocking) As far as I can tell there is no harm in making `t_range` a numpy array here -- but also no real benefit. (No numpy-specific methods are used (?) on `t_range` anywhere; all that is required is access to the components of the list.)

Is there any particular reason why you turned this into a numpy array?

Reply:
While i was looking for the source of the slowness bug, I tried to use `np.array()` in several areas of the code to see if it could net some kind of speed increase/fix. When i actually found the bug I probably forgot to revert the np.array's. So there's no real reason and I'm cool with it being reverted.

Reply:
Done: aa1bb0b2533359728b2eefe57b296d18c4e41aa7 -- could you briefly cross-check my changes?

Reply:
Looks good 👍